uswds/uswds-compile
# USWDS-Compile - POAM: July '24

# Summary

POAM for July 2024.

## Vulnerabilities

No changes in vulnerabilities

```node
3 moderate severity vulnerabilities
```

## Testing instructions

1. Install this branch on site
    - On Site's `main` branch, run: 
  ```bash
npm install "https://github.com/uswds/uswds-compile/tree/cm-POAM-july-2024" --save
```
2. Run gulp sass scripts and ensure compile runs as expected without error.
3. Ensure no regressions when package is used to compile site

## Dependency updates

| Dependency name | Previous version | Updated version |
|--------|--------|--------|
| postcss | ^8.4.38  | ^8.4.39 |
